Updated README

michaelsilver opened the pull request:
gdbm dependency and development install
closes #32

fakedrake commented:
Before I merge this just so we can be clear: `pip install -r requirements.txt` *should not* be required anymore. Since WikipediaBase is to be used as a library too from now on all dependencies should be maintained in setup.py. Setup tools will do the right thing and install any missing deps when you do `python setup.py {install,develop,...}`

Could you remove requirements.txt and the relevant documentation and check that it works correctly for you before I merge this.

fakedrake commented:
Before I merge this just so we can be clear: `pip install -r requirements.txt` *should not* be required anymore. Since WikipediaBase is to be used as a library too from now on all dependencies should be maintained in setup.py. Setup tools will do the right thing and install any missing deps when you do `python setup.py {install,develop,...}`

Could you remove requirements.txt and the relevant documentation and check that it works correctly for you before I merge this.

michaelsilver commented:
Sorry for the delay, @fakedrake. I've now checked and after adding `sqlitedict` to the `setup.py`, `python setup.py install` works without the `requirements.txt`, which I have now deleted. Updated the `README.md` per your requrest, and even included the caution message you wrote in #38.
